src/component/AddressForm.jsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { UseAddressContext } from "../context/address-context";
+
+export function AddressForm(disp,setDisp) {
+
+    const [name, setname] = useState("");
+    const [mobileNumber, setmobileNumber] = useState("");
+    const [pincode, setpinCode] = useState("");
+    const [Location, setLocation] = useState("");
+    const [city, setcity] = useState("");
+    const [Address, setAddress] = useState("");
+    const [state, setstate] = useState("");
```

Comment:
You can separate those setError,state to a context like AddressmanagementContext